advanced/multichain/rpc-reference/sui-rpc.mdx

```diff
@@ -79,7 +79,7 @@ Sign a Sui transaction without executing it.
   "jsonrpc": "2.0",
   "result": {
     "signature": "ACRvdr3yI2mdpeOK+NsJIimdNGcE9R//jjT3HALZ17fFyu818op4jZi/64lPBjpKMDX6ZtxnCFZExTOFdpi3MwEZXLv/ORduxMYX0fw8dbHlnWC8WG0ymrlAmARpEibbhw==",
```

Comment:
Let's change signature to: AGte9GqgPwHIzFSr/A4RdYqcgk2QJof0m8pHt06+WsIrw3vU40B+HGpQS/KaA9nPh12i/A7tIp6j5DGPoKM44AEZXLv/ORduxMYX0fw8dbHlnWC8WG0ymrlAmARpEibbhw==